content/blog/agentic-ai-in-agile-smarter-sprints-faster-retros.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Agentic AI in agile: Smarter sprints, faster retros"
+date: 2025-09-05T15:29:28.334Z
+author: Dinesh R Singh, Nisha Rajput, Varsha Shekhawat
+authorimage: /img/blog-image.jpeg
+disable: false
+tags:
+  - Agile
+  - Project manager
+  - Scrum master
+  - Agile coach
+---
+<style>
+
+li {
+
+   font-size: 27px;
+
+   line-height: 33px;
+
+   max-width: none;
+
+}
+
+</style>
+
+\*Part 5 of our **Gen AI for PM series***
```

Comment:
Please remove the "\\" character in front of *Part 5 of our **Gen AI for PM series***